client/ayon_core/hosts/houdini/plugins/create/create_mantra_rop.py

```diff
@@ -101,5 +95,24 @@ def get_pre_create_attr_defs(self):
                     label="Override Camera Resolution",
                     tooltip="Override the current camera "
                             "resolution, recommended for IPR.",
-                    default=False)
+                    default=False),
+            UISeparatorDef(key="1"),
+            UILabelDef(label="Farm Render Options:"),
+            BoolDef("farm",
+                    label="Submitting to Farm",
+                    default=True),
```

Comment:
Can we make this an `EnumDef` with options that say:
```
Local Render
Farm Render
```

That way we can expand it later to also include:
```
Local Render (Use Existing Frames)
Farm Render (Use Existing Frames)
```
Which is something I blelieve Nuke and Fusion also do.

Reply:
> Can we make this an `EnumDef` with options that say:
> 
> ```
> Local Render
> Farm Render
> ```
> 
> That way we can expand it later to also include:
> 
> ```
> Use Existing Frames (local)
> Use Existing Frames (on farm)
> ```
> 
> Which is something I blelieve Nuke and Fusion also do. Would love to hear other options though.
> 
> That way the `skip_render` toggle is basically part of this combox

we can take references from what the local/farm rendering setting in Nuke hosts. 
1. Local Render
2. Local Render(with existing frames)
3. Farm Render
It is more user-friendly especially when both local render with or without existing frames as the users can always manually choose whether they use the frames which they manually rendered.or not.

Reply:
implemented in https://github.com/ynput/ayon-core/pull/328/commits/bcb1c2a0ba3b49c43a08507ad52a9fdce07037ef